packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -505,7 +511,16 @@ class Center extends Align {
 /// decide where to position the child. The delegate can also determine the size
 /// of the parent, but the size of the parent cannot depend on the size of the
 /// child.
+///
+/// See also:
+///
+///  * [SingleChildLayoutDelegate]
+///  * [Align]
+///  * [CustomMultiChildLayout]
```

Comment:
` * [CustomMultiChildLayout], the same as [SingleChildCustomLayout] but for multiple children` or some such